src/VisualStudio/Core/Def/LanguageService/AbstractLanguageService`2.VsCodeWindowManager.cs

```diff
@@ -267,6 +275,12 @@ int IVsDocOutlineProvider.GetOutline(out IntPtr phwnd, out IOleCommandTarget? pp
 
             int IVsDocOutlineProvider.ReleaseOutline(IntPtr hwnd, IOleCommandTarget pCmdTarget)
             {
+                var enabled = _globalOptions.GetOption(DocumentOutlineOptionsMetadata.EnableDocumentOutline);
```

Comment:
What happens if the user changes the option after we created one? Wouldn't this then leak?

Reply:
Good point, we should just always dispose these if they are not null